src/coreComponents/physicsSolvers/CMakeLists.txt

```diff
@@ -7,233 +7,49 @@ set( physicsSolvers_headers
-     fluidFlow/CompositionalMultiphaseFVM.cpp
-     fluidFlow/CompositionalMultiphaseStatistics.cpp
-     fluidFlow/IsothermalCompositionalMultiphaseFVMKernels.cpp
-     fluidFlow/CompositionalMultiphaseHybridFVM.cpp
-     fluidFlow/CompositionalMultiphaseHybridFVMKernels.cpp
-     fluidFlow/ReactiveCompositionalMultiphaseOBL.cpp
-     fluidFlow/FlowSolverBase.cpp
-     fluidFlow/proppantTransport/ProppantTransport.cpp
-     fluidFlow/proppantTransport/ProppantTransportKernels.cpp
-     fluidFlow/SinglePhaseBase.cpp
-     fluidFlow/SinglePhaseStatistics.cpp
-     fluidFlow/SinglePhaseFVM.cpp
-     fluidFlow/SinglePhaseHybridFVM.cpp
-     fluidFlow/SinglePhaseProppantBase.cpp
-     fluidFlow/SinglePhaseProppantFluxKernels.cpp
-     fluidFlow/wells/CompositionalMultiphaseWell.cpp
-     fluidFlow/wells/CompositionalMultiphaseWellKernels.cpp
-     fluidFlow/wells/SinglePhaseWell.cpp
-     fluidFlow/wells/SinglePhaseWellKernels.cpp
-     fluidFlow/wells/WellControls.cpp
-     fluidFlow/wells/WellSolverBase.cpp
-     inducedSeismicity/SeismicityRate.cpp
-     multiphysics/CompositionalMultiphaseReservoirAndWells.cpp
-     multiphysics/CoupledReservoirAndWellsBase.cpp
-     multiphysics/FlowProppantTransportSolver.cpp
-     multiphysics/HydrofractureSolver.cpp
-     multiphysics/MultiphasePoromechanics.cpp
-     multiphysics/PhaseFieldFractureSolver.cpp
-     multiphysics/PoromechanicsInitialization.cpp
-     multiphysics/SinglePhasePoromechanics.cpp
-     multiphysics/SinglePhasePoromechanicsEmbeddedFractures.cpp
-     multiphysics/SinglePhasePoromechanicsConformingFractures.cpp
-     multiphysics/SinglePhaseReservoirAndWells.cpp
-     simplePDE/LaplaceBaseH1.cpp
-     simplePDE/LaplaceFEM.cpp
-     simplePDE/PhaseFieldDamageFEM.cpp
-     solidMechanics/SolidMechanicsLagrangianFEM.cpp
-     solidMechanics/SolidMechanicsLagrangianSSLE.cpp
-     solidMechanics/SolidMechanicsMPM.cpp
-     solidMechanics/SolidMechanicsStateReset.cpp
-     solidMechanics/SolidMechanicsStatistics.cpp
-     surfaceGeneration/EmbeddedSurfaceGenerator.cpp
-     surfaceGeneration/EmbeddedSurfacesParallelSynchronization.cpp
-     surfaceGeneration/ParallelTopologyChange.cpp
-     surfaceGeneration/SurfaceGenerator.cpp
-     wavePropagation/shared/WaveSolverBase.cpp
-     wavePropagation/sem/acoustic/secondOrderEqn/isotropic/AcousticWaveEquationSEM.cpp
-     wavePropagation/sem/elastic/secondOrderEqn/isotropic/ElasticWaveEquationSEM.cpp
-     wavePropagation/sem/elastic/firstOrderEqn/isotropic/ElasticFirstOrderWaveEquationSEM.cpp
-     wavePropagation/sem/acoustic/firstOrderEqn/isotropic/AcousticFirstOrderWaveEquationSEM.cpp
-     wavePropagation/sem/acoustic/secondOrderEqn/anisotropic/AcousticVTIWaveEquationSEM.cpp
-     wavePropagation/sem/acoustoelastic/secondOrderEqn/isotropic/AcousticElasticWaveEquationSEM.cpp )
+     SolverStatistics.cpp )
 
-  include( solidMechanics/kernels/SolidMechanicsKernels.cmake)
+if( GEOS_ENABLE_CONTACT )
+  add_subdirectory( contact )
```

Comment:
I can move contactmechanics kernels to their own directory too.

Reply:
did I miss those?